docs/content/how-to/writing-tables.md

```diff
@@ -316,6 +318,84 @@ For more information of 'delete', see
 
 {{< /tabs >}}
 
+
+### Deleting from table with Flink version >= 1.17
```

Comment:
I think we can keep single section. Just split to two tabs.

Reply:
done